### ISSUE TITLE
Increase the number of themes available

### ISSUE THREAD

**Issue by James-Oldfield** [thumbsup/node-thumbsup#37]
Hi there! Fantastic little module, thanks ever so much 😁! 

It would be lovely to have a small range of other themes (viz. a few other `.css`) that can be loaded in place of the original default one. What are your thoughts on this?

Cheers! Again, great job!

James.

**Comment by rprieto** [comment]
Hi, sorry just seeing this issue now. That's a great idea!

Do you think CSS is enough, or should it include the `hbs` templates too? It would be more flexible for designs, but also more work to maintain if the data model changes.

Do you have some designs to suggest for the first themes?

**Comment by rprieto** [comment]
Quick update, PR #44 makes this a lot easier. There is now a well-defined and unit-tested canonical model for Albums, and rendering is simplified (single page).

Next step, I'll group the current theme into a folder that looks like this:

```
themes 
    |__ classic
        |__ theme.hbs
        |__ theme.css
ˋ``

From there it should be easy to create new themes.

**Comment by rprieto** [comment]
Hi @James-Oldfield, in case you're still interested: I just released preview version `2.0.0-alpha.1` which supports 3 different themes:

## Classic

![thumbsup classic](https://cloud.githubusercontent.com/assets/189105/22330185/0d9c903a-e419-11e6-93a3-12b437c8481b.png)

## Cards

![thumbsup cards](https://cloud.githubusercontent.com/assets/189105/22330188/1181643c-e419-11e6-9dfb-339eb767e7d7.png)

## Mosaic

![thumbsup mosaic](https://cloud.githubusercontent.com/assets/189105/22330186/0fb5b5e0-e419-11e6-9920-5b9e34097873.png)
